### DISCUSSION TITLE
How to use _id in Data Provider filters

### DISCUSSION THREAD

**Discussion by tstauch** [Budibase/budibase#4726] [Ideas]
The auto-generated screens for DB tables include a filter on the Data Provider:

_id Equals Binding {{ URL.id }}

I am working with composite keys, e.g. [id,published] 
If I try to create my own screen with the same filter, _id is not available in the drop down list for Define Filters.
This means I end up having to create non-standard routes, 
e.g .../:id/:published

Any plans to make _id available in the Filtering rules?

**Comment by mjashanks** [comment]
Hey @tstauch thanks for this - we don't plan to support `_id` filtering for composite keys soon - as you've pointed, there is a workaround. This de-prioritises it somewhat.

As I think you pointed out elsewhere, you could also send the URL encoded format to `../:id` ... like `%5B10%2C1%5D` - then parse that in Javascript.
